sdk/purview/Azure.Analytics.Purview.Scanning/src/Generated/PurviewClassificationRuleClient.cs

```diff
@@ -71,6 +71,20 @@ public PurviewClassificationRuleClient(Uri endpoint, string classificationRuleNa
 
         /// <summary> Get a classification rule. </summary>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <example>
```

Comment:
It looks like this one only has one example, not one for required and one for all properties.  Can you help remind me why?

Reply:
Because in the [original swagger spec](https://github.com/Azure/azure-rest-api-specs/blob/1c7df99f6a84335cfd7bf5be8c800d72c1dddbc2/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json#L197-L236), this method has one required parameter only, so here we just generate one example. And in this example, we parse all properties of the [response](https://github.com/Azure/azure-rest-api-specs/blob/1c7df99f6a84335cfd7bf5be8c800d72c1dddbc2/specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2018-12-01-preview/scanningService.json#L1926-L1950).

Reply:
Thank you!